flask_resty/view.py

```diff
@@ -258,6 +258,13 @@ def delete_item(self, item):
 
         self.session.delete(item)
 
+    def flush(self):
+        try:
+            # Flushing allows checking invariants without committing.
+            self.session.flush()
+        except IntegrityError:
+            raise ApiError(409, {'code': 'invalid_data.conflict'})
```

Comment:
not an easy way to do so – it's small enough that it's not hugely worthwhile to pull that out into a method just to raise an exception imo